dist/scripts/api-console.js

```diff
@@ -785,30 +789,33 @@
       controller: ['$scope', '$window', function($scope, $window) {
         $scope.ramlUrl    = '';
 
-        ramlParserWrapper.onParseError(function(error) {
-          /*jshint camelcase: false */
-          var context = error.context_mark || error.problem_mark;
-          /*jshint camelcase: true */
-
-          $scope.errorMessage = error.message;
+        ramlParserWrapper.onParseError(function(errors) {
+          $scope.errors = errors.parserErrors.map(function (theError) {
+            return {
+              errorMessage: theError.message,
+              line: theError.line,
+              column: theError.column
+            };
+          });
 
-          if (context && !$scope.isLoadedFromUrl) {
-            $scope.raml = context.buffer;
+          if (!$scope.loadFromUrl) {
+            var editor = jQuery('.raml-console-initializer-input-container .CodeMirror')[0].CodeMirror;
 
-            $window.ramlErrors.line    = context.line;
-            $window.ramlErrors.message = error.message;
+            $scope.errors.forEach(function (theError) {
```

Comment:
I guess that `errors` has multiple, but using `error` is clearer, otherwise I expect a single error to exist in the whole context, which exactly the opposite.